### PR TITLE
[cf]-added-destinations-back

### PR DESCRIPTION
**Pull request by colefreeman**
# Description
Added destinations back to redirects for streaming links

# How Has This Been Tested?
Tested in mitlify dev env

- [ ] Test A
- [ ] Test B

# Checklist
- [X] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [X] I have performed a self-review of my own code
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X
Screenshot 2024-12-13 at 4 12 31 PM
Screenshot 2024-12-13 at 4 14 45 PM
Screenshot 2024-12-13 at 4 15 04 PM
] I have made corresponding changes to the documentation

cc:
@johnson-mage

### CODE REVIEW

**Comment by johnson-mage** [comment]
@colefreeman what happens if you just remove those redirects? Cuz redirecting to itself seems unnecessary? The source and destination are all the same.

**Comment by colefreeman** [comment]
@johnson-mage removed redirects for streaming sources and destinations 
Screenshot 2024-12-13 at 5 09 32 PM
Screenshot 2024-12-13 at 5 09 44 PM

**Comment by johnson-mage** [comment]
@colefreeman Is this being used anywhere? If not, please remove, but make sure you check the docs to see if it's used anywhere first.
```
{
      "source": "/guides/streaming/overview",
      "destination": "/streaming/overview"
    },
```

**Comment by colefreeman** [comment]
@johnson-mage added back links, they are working in the dev env

File: docs/mint.json [hunk 1]
```diff
@@ -368,51 +368,96 @@
...
     },
     {
-      "source": "/guides/streaming/sources/kafka"
+      "source": "/streaming/sources/kinesis",
+      "destination": "/guides/streaming/sources/kinesis"
     },
     {
-      "source": "/guides/streaming/sources/kinesis"
+      "source": "/guides/streaming-pipeline-rabbitmq",
+      "destination": "/guides/streaming/sources/rabbitmq"
```
Comment (johnson-mage) [inline docs/mint.json:460]: @colefreeman You should search for a doc named `streaming-pipeline-rabbitmq.mdx` instead. To keep the reference consistent, the destination should reference this doc: `docs/guides/streaming/tutorials/streaming-pipeline-rabbitmq.mdx`. We have an actual guide for rabbitmq specifically.

**Comment by colefreeman** [comment]
@johnson-mage removed streaming overview, did not find it anywhere in the docs

**Review by johnson-mage — APPROVED** [review]

**Pull request merged**